osidb/models.py

```diff
@@ -750,6 +750,24 @@ def _validate_future_unembargo_date(self):
                 "Flaw still embargoed but unembargo date is in the past."
             )
 
+    def _validate_public_flaw_summary(self):
+        """
+        Check that the flaw's summary does not contain the word "EMBARGOED" if the flaw is public.
+        """
+        if not self.is_embargoed and "EMBARGOED" in self.summary:
```

Comment:
There is one tricky thing in OSIDB. It is not very easy to spot but you can see a comment at https://github.com/RedHatProductSecurity/osidb/blob/master/osidb/models.py#L357 that what we used to call summary in the SFM2 terminology (and in Bugzilla) is actually called **title** in SFM2. The summary here needs to be replaced by title

Reply:
And not just here. Basically everywhere in the PR you need to use **title**

Reply:
Refactored the validation + tests to reflect the story correctly.